docs/source/library_reference.rst

```diff
@@ -195,11 +195,13 @@ Discovering Tests
 .. envvar:: COCOTB_RESULTS_FILE
 
     The file name where xUnit XML tests results are stored. If not provided, the default is :file:`results.xml`.
-    Generated XML file is fully compatible with xUnit version 1.x (``xunit1``) and 2.x (``xunit2``).
-    The default xUnit format is ``xunit2``. The ``xunit1`` format is used in GitLab CI environments.
 
     .. versionadded:: 1.3
 
+    .. versionchanged:: 2.1
+        Generated XML file is fully compatible with xUnit version 1.x (`xunit1`_) and 2.x (`xunit2`_).
+        The default xUnit format is `xunit2`_. The `xunit1`_ format is used in GitLab CI environments.
```

Comment:
I'm confused -- the output is compatible with both schemas? How does it then matter what GitLab CI "uses" (as in, can consume?)

Reply:
@imphil Bad wording. I will "refactor" it. Please refer to my comment https://github.com/cocotb/cocotb/pull/5219#issuecomment-3694255165

GitLab is happy with both versions but with v1 it can use `file` attribute to link source file with test in their tests report: [GitLab JUnit format specification](https://docs.gitlab.com/ci/testing/unit_test_reports/#junit-xml-format-specification). Where Jenkins is unhappy with v1.